code/components/citizen-server-impl/src/InitConnectMethod.cpp

```diff
@@ -564,7 +565,7 @@ static InitFunction initFunction([]()
 			auto gameBuildField = (gameBuildIt != postMap.end()) ? gameBuildIt->second : "0";
 			auto gameName = (gameNameIt != postMap.end()) ? gameNameIt->second : "";
 
-			if (protocol < 12)
+			if (protocol < kNetworkProtocolVersion)
```

Comment:
Isn't this breaking the compatibility when the current clients still send 12, because of the NETWORK_PROTOCOL macro?

Reply:
Yes, I put the wrong protocol number here, will fix it when I get home this should be 12 and 13